wp-content/plugins/core/src/Blocks/Block_Category.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Blocks;
+
+/**
+ * Class Block_Category
+ *
+ * Edits the block categories.
+ */
+class Block_Category {
+
+	/**
+	 * Adds the Custom Category to the front of the block list categories.
+	 *
+	 * @param array $categories
+	 *
+	 * @return array
+	 */
+	public function custom_block_category( array $categories ): array {
+		return array_merge( [
+			[
+				'slug'  => 'tribe-custom',
```

Comment:
We should make this a constant in this class so it can be used across the other blocks and also updated in a single place so a developer doesn't have to edit every single block after this if we need to change the slug.

Reply:
Done.